src/main/java/bio/terra/testrunner/runner/config/ClusterSpecification.java

```diff
@@ -13,6 +13,9 @@
   public String project;
   public String namespace;
   public String containerName;
+  // Default metadata labels to look up service deployment in k8s.
```

Comment:
Does this need to be updated in the README also?

Reply:
Updated README with definitions of `componentLabel` and `apiComponentLabel`.